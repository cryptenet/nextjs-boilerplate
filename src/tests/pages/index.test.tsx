import IndexPage from '@pages/index';
import { render } from '../testUtils';

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<IndexPage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
