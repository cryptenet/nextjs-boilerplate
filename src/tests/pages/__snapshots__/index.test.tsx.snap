// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="c-EeDqC"
  >
    <header
      class="c-fziZbc"
    >
      <nav
        class="c-PJLV"
      />
    </header>
    <section
      class="c-jJSOgF"
    >
      <div
        class="c-gTpyGt"
      />
      <p
        class="c-PJLV c-PJLV-ijiKEYY-css"
      >
        NextJS Boilerplate
      </p>
    </section>
    <footer
      class="c-jOkXbd"
    >
      <p>
        Â© Cryptenet Limited 2021
      </p>
    </footer>
  </main>
</DocumentFragment>
`;
